data-prepper-plugins/elasticsearch/README.md

```diff
@@ -42,47 +42,18 @@ pipeline:
 
 The elasticsearch sink will reserve `otel-v1-apm-service-map` as index for record ingestion.
 
-### Custom data
-
-```$xslt
-pipeline:
-  ...
-  sink:
-    elasticsearch:
-      hosts: ["https://localhost:9200"]
-      username: YOUR_USERNAME_HERE
-      password: YOUR_PASSWORD_HERE
-      index: "some-index"
-      template_file: /your/local/template-file.json
-      document_id_field: "someId"
-      dlq_file: /your/local/dlq-file
-      bulk_size: 4
-```
-
-User needs to provide custom index for record ingestion.
-
-## Parameters
-
-### Hosts
+## Configuration
 
-A list of IP addresses of elasticsearch nodes.
+- `hosts`: A list of IP addresses of elasticsearch nodes.
 
-### cert
-CA certificate that is pem encoded. Accepts both .pem or .crt. This enables the client to trust the CA that has signed the certificate that ODFE is using.
+- `cert`(optional): CA certificate that is pem encoded. Accepts both .pem or .crt. This enables the client to trust the CA that has signed the certificate that ODFE is using.
 Default is null. 
 
-### Username
+- `username`(optional): A String of username used in the [cognito](https://opendistro.github.io/for-elasticsearch-docs/docs/security/access-control/users-roles) of ODFE cluster. Default is null.
 
-A String of username used in the [cognito](https://opendistro.github.io/for-elasticsearch-docs/docs/security/access-control/users-roles) of ODFE cluster.
-
-### Password
-
-A String of password used in the [cognito](https://opendistro.github.io/for-elasticsearch-docs/docs/security/access-control/users-roles) of ODFE cluster.
-
-### Trace analytics raw
-
-A boolean flag indicates APM trace analytics raw span data type. e.g.
+- `password`(optional): A String of password used in the [cognito](https://opendistro.github.io/for-elasticsearch-docs/docs/security/access-control/users-roles) of ODFE cluster. Default is null.
 
+- `trace_analytics_raw`(optional): A boolean flag indicates APM trace analytics raw span data type. e.g.
```

Comment:
How does setting true or false affect behavior, i.e. when should I override the default value of false? (I know the answer personally, but I don't think it seems obvious to a new user).

Same comment applies to `trace_analytics_service_map` below.

Reply:
Will add an additional comment inline.